Gemfile

```diff
@@ -72,7 +72,10 @@ end
 # end
 
 group :development, :test do
+  gem 'capybara'
```

Comment:
**You both did great job!** 

- [ ] However, as the previous reviewer suggested we have failing tests. Kindly, fix them before submitting the. Maybe check your db/seeds.rb file and add some preload users, posts, comments, likes... I can see you did added but the issue is still appearing. This is a video referace kindly watch it!


https://user-images.githubusercontent.com/69011963/216124154-cdd459bd-5bec-4c83-8649-0088b5a0e3ec.mov

_Try to recloen your project and reset your database then test your code. I think with this try you will see the errors._

Reply:
@Rustamxon7 I'm so sorry, I didn't push the changes